kythe/typescript/testdata/function.ts

```diff
@@ -32,6 +32,21 @@ function test(a: number, num: Num): Num {
 //- @test ref F
 test(3, {num: 3});
 
+
+// Test comoputed function names
```

Comment:
```suggestion
// Test computed function names
```

It may also be worth explaining _how_ this tests computed function names.